frontend/ui/widget/dictquicklookup.lua

```diff
@@ -104,8 +104,8 @@ function DictQuickLookup:init()
         font_size_alt = 8
     end
     self.image_alt_face = Font:getFace("cfont", font_size_alt)
-    self.allow_text_selection = Device:hasScreenKB() or Device:hasKeyboard()
-    if self.allow_text_selection then
+    self.allow_nt_text_selection = Device:hasScreenKB() or Device:hasKeyboard()
```

Comment:
If it's not `not IsTouchDevice()`, don't name it "nt_text_selection"...
Name it "key_text_selection".